nginx/modsecurity/ngx_http_modsecurity.c

```diff
@@ -635,6 +648,10 @@ ngx_http_modsecurity_init(ngx_conf_t *cf)
     }
 #endif    
 
+    azwaf_processing_result_key = ngx_hash_key_lc(
```

Comment:
what's the usage of the key here, just to look up the variable when receive the request?

Reply:
Yes, it is required by the lookup function.